apps/storybook-e2e/src/e2e/map/action/click-action/vl-map-click-action.stories-dom.cy.ts

```diff
@@ -0,0 +1,39 @@
+describe('story vl-map-select-actions default', () => {
+    const mapClickActionUrl =
+        'http://localhost:8080/iframe.html?id=map-action-click-action--map-click-action-default&viewMode=story';
+
+    function getMap() {
+        return cy.get('vl-map');
```

Comment:
Dit doen we normaal gezien niet omdat dit niet veel toegevoegde waarde heeft.
getMap() is niet veel minder karakters dan cy.get('vl-map') en maakt het enkel complexer.

Reply:
opgelost